### DISCUSSION TITLE
Error while placing Order - Payment Service

### DISCUSSION THREAD

**Discussion by naveen3607** [learndevopsonline/learndevopsonline#1497] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/138672193/7e2eda7c-3e85-4888-a947-87542e4828d6)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/138672193/ef15c98c-98d0-4d33-aa6f-bc7c9b4c24d4)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/138672193/6ee644f4-7768-41d3-b6e3-7e2529cce3f3)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/138672193/d6f8ad0d-12d8-499b-a900-462b17971c0e)

Hi Raghu,

I created a complete infrastructure using Terraform, Ansible, Jenkins and Nexus. Every component is working fine except the Payment service. I tried to restart the service many times, but it is not becoming active. Can you please guide me on how to rectify this error?

Thanks and Regards,
Naveen Reddy Kallam

**Comment by sreedharm07** [comment]
Hi @naveen3607 can you please share your ip if the error is still exists

**Reply by naveen3607** [reply]
@sreedharm07 workstation ip: 18.208.133.178

**Reply by sreedharm07** [reply]
Hi @naveen3607 
you have not installed dependencies, please download them, in your ansible code you have disabled the dependencies code

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/535740ac-f5f4-4847-a03c-e71ad56ae500)

**Reply by naveen3607** [reply]
@sreedharm07, did you run an ansible pull in my payment server?

**Reply by sreedharm07** [reply]
I dint run ansible pull but directly executed dependencies in your payment server

**Reply by naveen3607** [reply]
I gave executable permission for uwsgi file. Then the payment service started running.
